docs-src/cloud/pricing-support.md

```diff
@@ -10,7 +10,7 @@ tags:
   - explanation
 ---
 
-With a subscription to Temporal Cloud, you also gain access to our support organization of developer success engineers and solution architects. Our experts assist with a range of work streams from Workflow design reviews to setting up observability to break/fix support with an agreed-upon set of SLAs.
+When signing up for Temporal Cloud, you select a support plan—either Basic or Premium—and gain access to our support organization of developer success engineers and solution architects. Our experts assist with a range of work streams from Workflow design reviews to setting up observability to break/fix support with an agreed-upon set of SLAs.
```

Comment:
```suggestion
When signing up for Temporal Cloud, select a support plan—either Basic or Premium—and gain access to our support organization, which includes developer success engineers and solution architects. Our experts assist with a range of work streams, from Workflow design reviews to setting up observability, and from break/fix support to maintaining an agreed-upon set of SLAs.
```

Reply:
I know you didn't write it, but I really don't like the `break/fix` line. Would troubleshooting be better?
We aren't dividing break by fix.